lib/src/components/NxCombobox/NxCombobox.tsx

```diff
@@ -122,7 +126,7 @@ function NxComboboxRender<T extends string | number | DataItem<string | number,
   function handleOnChange(newVal: string) {
     setFocusableBtnIndex(null);
     onChange(newVal);
-
+    setHiddenBySelection(false);
```

Comment:
Is there any case that this would execute and `hiddenBySelection` not already be false? This is essentially the `onChange` handler of the `<input>`, which I think could only fire if the `<input>` has focus, and it getting focus should already set `hiddenBySelection` to false, right?

Reply:
> Is there any case that this would execute and `hiddenBySelection` not already be false? This is essentially the `onChange` handler of the `<input>`, which I think could only fire if the `<input>` has focus, and it getting focus should already set `hiddenBySelection` to false, right?

This was kept for the scenario that a selection was made (closing the dropdown), and then the user editing the `<input>` value without shifting focus outside of `NxCombobox`